### PR TITLE
Add functionality to movie page

### PR DESCRIPTION
**Pull request by Kookiie88**
This update introduces both POST and GET routes for reviews, along with the capability to post your own reviews directly on individual movie pages. Additionally, you can now view all reviews associated with each movie.

To post a review, simply navigate to the desired movie page and submit your review. This will add your feedback to the collection of reviews for that specific movie.

With the GET route implemented, you can easily retrieve all reviews for any given movie, enhancing the overall user experience by providing valuable insights and feedback.

This feature enriches our platform by fostering interaction and engagement among users, while also facilitating informed decision-making for movie enthusiasts.

### CODE REVIEW

File: kinogoteborg/src/app/components/movie/Reviews.jsx [hunk 1]
```diff
@@ -18,64 +28,31 @@ export default function Reviews({ isVisible, onClose, setShowReviewForm, setShow
...
-                <li>
-                  <span className="mr-2">Rating:</span> 4/5{" "}
-                </li>
-                <li>
-                  <span className="mr-2">Review:</span> Lorem ipsum dolor sit amet consectetur
-                  adipisicing elit. Quisquam, quos?
-                </li>
-              </ul>
-            </div>
+            {reviews.map((review) => {
```
Comment (AnnaLenaO) [inline kinogoteborg/src/app/components/movie/Reviews.jsx:31]: Nice changes made.

**Review by AnnaLenaO — COMMENTED** [review]
Please se PR Create admin Post routes #51 for a comment about the file screeningModel.js that also is included in this PR.

**Comment by Kookiie88** [comment]
This pull request is linked to PBI #53 #54

File: kinogoteborg/src/app/components/movie/ReviewForm.jsx [hunk 2]
```diff
@@ -12,19 +46,45 @@ export default function Reviews({ isVisible, onClose }) {
         </button>
         <div className=" bg-stone-800 p-2 rounded-md text-stone-200">
           <h2 className="text-2xl text-center">Write a review of your own</h2>
-          <form action="submit" className="flex flex-col gap-2">
+          <form className="flex flex-col gap-2" onSubmit={submitReview}>
             <label htmlFor="Author">Author</label>
```
Comment (AnnaLenaO) [inline kinogoteborg/src/app/components/movie/ReviewForm.jsx:50]: I am not sure if it is supposed to be `<label htmlFor=...>` or if the last "m" is missed in `<label htmlForm=...>`.

**Review by AnnaLenaO — APPROVED** [review]

**Pull request merged**